scripts/leapfrog/README.md

```diff
@@ -41,6 +41,15 @@ page](https://github.com/rcbops/rpc-openstack/tree/r14.1.0) shows both SHAs,
 the RPCO SHA is the latest commit, and the OSA SHA is shown as an annotation
 after @ on the openstack-ansible submodule.
 
+Two variables will need to be set in `/etc/openstack_deploy/user_variabls.yml`
+before proceeding with the upgrade
+
+```
+lxc_container_backing_store: "dir"
```

Comment:
Alternatively deployers could set 'lvm' (or even 'overlayfs'). Our tested code is "dir".  

Maybe worth saying:

> ``lxc_container_backing_store: "dir"`` or ``lxc_container_backing_store: "lvm"`` depending on your storage.

(I don't think we've ever supported overlayfs)